webapp/publisher/snaps/views.py

```diff
@@ -286,9 +290,9 @@ def redirect_get_register_name():
 @publisher_snaps.route("/register-snap")
 @login_required
 def get_register_name():
-    user = publisher_api.get_account(flask.session)
+    stores = admin_api.get_stores(flask.session)
```

Comment:
The account no longer has the list of stores, so get the list of stores directly from the API